### ISSUE TITLE
Is it possible to increase the video socket limit?

### ISSUE THREAD

**Issue by brynrichards** [microsoftgraph/microsoft-graph-comms-samples#649]
Would it be possible to increase the video socket limit from 10?

The CPU overhead is not a concern for my use case.

**Comment by ssulzer** [comment]
@brynrichards 
No, the limit is 10 VideoSockets for main (webcam) video per bot call session.
May I ask what is your intended use case that would require more? How many VideoSockets would your scenario typically require? Thanks.

**Comment by brynrichards** [comment]
@ssulzer thanks for confirming.

We have a 3d immersive events environment where we display > 100 participant web cams received via WebRTC using a browser, with the H264 being decoded on a GPU.   We would like to be able to let participants join an event via teams, and I've been able to forward 10 incoming H264 video streams directly from the bot with low overhead, however if there's a hard limit of 10 I think I would need to have some sort of coordination layer and multiple instances of the bot joining and subscribing to different streams.

**Comment by InDieTasten** [comment]
@brynrichards Not sure if your use case would be covered by this, but you could also operate a single instance that subscribes and unsubscribes based on who is talking in the meeting and keep to the limit of 10, just like the Teams client does for example.
